### PR TITLE
#67 development server good server should be configured to work with adminloc

### PR DESCRIPTION
**Pull request by StanislavShevchenko**
#67  **development server "good-server" should be configured to work with adminlock**

 `
The branch is branched off from branch [#395 Server private gundb should be replaced with mongodb](https://github.com/GoodDollar/GoodServer/tree/%23395-Server-private-gundb-should-be-replaced-with-mongodb)
`

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -41,3 +41,4 @@ ROLLBAR_TOKEN=
 MONGO_DB_URI=
 WEB3_SITE_URL=
 DISABLE_FACE_VERIFICATION=
+ENABLE_MONGO_LOCK=false
```
Comment (yaroslav-fedyshyn-nordwhale) [inline .env.example]: @StanislavShevchenko no need to put values for variables in .env.example file

File: src/server/blockchain/__tests__/adminWallet.test.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import AdminWallet from '../AdminWallet'
 
-jest.setTimeout(30000)
+jest.setTimeout(3000000)
```
Comment (yaroslav-fedyshyn-nordwhale) [inline src/server/blockchain/__tests__/adminWallet.test.js]: @StanislavShevchenko Does it really need so much time to wait for the test to be done?

File: src/server/utils/tx-manager/queueMongo.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    try {
+      let wallet = await this.model.findOneAndUpdate(
+        { address, isLock: false },
+        { isLock: true },
+        { returnNewDocument: true }
+      )
+
+      return wallet
+    } catch (e) {
+      console.log(e)
```
Comment (yaroslav-fedyshyn-nordwhale) [inline src/server/utils/tx-manager/queueMongo.js]: @StanislavShevchenko log error with logger

File: src/server/utils/tx-manager/queueMongo.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+      let wallet = await this.model.findOne({ address })
+
+      if (!wallet) {
+        await this.model.create({
+          address,
+          nonce: netNonce
+        })
+      }
+    } catch (e) {
+      console.log(e)
```
Comment (yaroslav-fedyshyn-nordwhale) [inline src/server/utils/tx-manager/queueMongo.js]: @StanislavShevchenko logger

File: src/server/utils/tx-manager/queueMongo.js [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+        const walletNonce = await this.getWalletNonce(nextTr.address)
+
+        if (walletNonce) {
+          this.queue.shift()
+          nextTr.cb(walletNonce.nonce)
+        }
+      }
+    } catch (e) {
+      console.log(e)
```
Comment (yaroslav-fedyshyn-nordwhale) [inline src/server/utils/tx-manager/queueMongo.js]: @StanislavShevchenko logger

File: .env.example (hunk 1 above)
Comment (StanislavShevchenko) [reply .env.example]: [done](https://github.com/GoodDollar/GoodServer/pull/68/commits/1d02ca129b9addb3145466fe576deffbb5068ac0)

File: src/server/blockchain/__tests__/adminWallet.test.js (hunk 2 above)
Comment (StanislavShevchenko) [reply src/server/blockchain/__tests__/adminWallet.test.js]: [done](https://github.com/GoodDollar/GoodServer/pull/68/commits/1d02ca129b9addb3145466fe576deffbb5068ac0)

File: src/server/utils/tx-manager/queueMongo.js (hunk 3 above)
Comment (StanislavShevchenko) [reply src/server/utils/tx-manager/queueMongo.js]: [done](https://github.com/GoodDollar/GoodServer/pull/68/commits/1d02ca129b9addb3145466fe576deffbb5068ac0)

File: src/server/utils/tx-manager/queueMongo.js (hunk 5 above)
Comment (StanislavShevchenko) [reply src/server/utils/tx-manager/queueMongo.js]: [done](https://github.com/GoodDollar/GoodServer/pull/68/commits/1d02ca129b9addb3145466fe576deffbb5068ac0)

File: src/server/utils/tx-manager/queueMongo.js (hunk 4 above)
Comment (StanislavShevchenko) [reply src/server/utils/tx-manager/queueMongo.js]: [done](https://github.com/GoodDollar/GoodServer/pull/68/commits/1d02ca129b9addb3145466fe576deffbb5068ac0)

**Pull request merged**